### PR TITLE
TomcatWork

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

**Review by mkharkhuta — APPROVED** [review]
a lot of unnecessary files, but the fact is that the housework is done well)

File: Homework.iml [hunk 1]
```diff
@@ -1,2 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4" />
\ No newline at end of file
+<module version="4">
```
Comment (SerhiiPasko) [inline Homework.iml:2]: I many times spoke about that. No any idea config must present in git repository

File: pom.xml [hunk 2]
```diff
@@ -17,34 +18,64 @@
 
     <build>
         <plugins>
+            <!--<plugin>-->
```
Comment (SerhiiPasko) [inline pom.xml:21]: Also

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments. Main task  - good

**Pull request merged**